spec.md

```diff
@@ -1081,6 +1092,34 @@ $instance = new class extends \Foo implements
 };
 ```
 
+## 9. Enumerations
+
+Enumerations (enums) MUST follow the same guidelines as classes, except where otherwise noted below.
+
+Methods in enums MUST follow the same guidelines as methods in classes.  Non-public methods MUST use `private`
+instead of `protected`, as enums do not support inheritance.
+
+When using a backed enum, there MUST NOT be a space between the enum name and colon, and there MUST be exactly one
+space between the colon and the backing type.  This is consistent with the style for return types.
+
+Enum case declarations MUST use CamelCase capitalization.  Enum case declarations MUST be on their own line.
+
+Constants in Enumerations MAY use either CamelCase or UPPER_CASE capitalization.  CamelCase is RECOMMENDED,
+so that it is consistent with case declarations.
```

Comment:
```suggestion
Enum case declarations MUST use CamelCase capitalization.  Enum case declarations MUST be on their own line.

Constants in Enumerations MAY use either PascalCase or UPPER_CASE capitalization. PascalCase is RECOMMENDED,
so that it is consistent with case declarations.
```